sqlx-core/src/postgres/types/mod.rs

```diff
@@ -72,6 +72,7 @@
 //! | Rust type                             | Postgres type(s)                                     |
 //! |---------------------------------------|------------------------------------------------------|
 //! | `ipnetwork::IpNetwork`                | INET, CIDR                                           |
+//! | `std::net::IpAddr`                    | INET, CIDR                                           |
```

Comment:
```suggestion
//! | `std::net::IpAddr`                    | INET, CIDR                                           |
//!
//! Note that because `IpAddr` does not support network prefixes, it is an error to decode an `INET` or `CIDR` value
//! with a network prefix smaller than the address' full width: `/32` for IPv4 addresses and `/128` for IPv6 addresses.
```

It's probably important to mention that.